### DISCUSSION TITLE
ImageMagick missing WebP

### DISCUSSION THREAD

**Discussion by mindsunday** [ImageMagick/ImageMagick#3644] [Help]
Hi,  I have CentOS 7.9.16 server and ImageMagick version 7.0.11. The problem is, WebP seems to be missing from the formats in PHP info. I have tried all possible solutions I found online with no success. Does anyone have any idea what I should do? My php version is PHP 7.4.16.

**Comment by fmw42** [comment]
My understanding is that you need to install WebP delegate library and then reinstall ImageMagick.
